pkg/configuration/configuration_test.go

```diff
@@ -81,19 +86,25 @@ func TestRegistrationService(t *testing.T) {
 			Verification().AWSRegion("us-west-2").
 			Verification().AWSSenderID("sandbox").
 			Verification().AWSSMSType("Transactional").
+			Verification().CaptchaEnabled(true).
+			Verification().CaptchaProjectID("test-project").
+			Verification().CaptchaSiteKey("site-key").
+			Verification().CaptchaScoreThreshold("0.7").
```

Comment:
Let's also test the default threshold: when it's not set then we use the default "0.9".

Reply:
We test the default on line 65 above. Or do you mean something else?

Reply:
Yeah I missed that.